project_national/tests/test.rb

```diff
@@ -4,7 +4,7 @@
 load 'Rakefile'
 require 'json'
 
-class TestProjectMultifamilyBeta < MiniTest::Test
+class TestProjectNational < MiniTest::Test
```

Comment:
Good catch.

Reply:
> This looks pretty straightforward.
> 
> Should the changelog be updated?
> 
> Have you tested these changes in an (large scale) analysis?
> 
> How are the png graphics produced? Looks like they are stored in the companion PR. Are you also storing whatever produces them?

@joseph-robertson:  Thanks for the review.

**_Should the changelog be updated?_**
I updated the changelog `902b77a`. 

**_Have you tested these changes in an (large scale) analysis?_**
We ran a national scale analysis (`res_national_21_01_01_2018 ` table, bucket enduse, on Athena). We did some comparisons to our AMI datasets.

**_How are the png graphics produced? Looks like they are stored in the companion PR. Are you also storing whatever produces them?_**
The graphics are actually being produced by QGIS, which is not scriptable. QGIS does, however, provide high-resolution images. I will see what I can commit to the `resstock-estimation` repository.